extend/component/code-patterns/overview.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Code Pattern Overview
+permalink: /extend/component/code-patterns/overview
+---
+
+* TOC
+{:toc}
+
+Code Pattern is a type of the [Component](/extend/component/).
+It can be used in the user interface of the [New Transformations](https://help.keboola.com/transformations/#new-transformations)
+to generate code based on the specified [Configuration](#configuration).
+
+
+## List of the Code Patterns
+
+In the [Keboola Developer Portal](https://components.keboola.com/components) can be found
+[published](/extend/publish/) components with the **Code Pattern type**.
+In the description you can find out for which transformation types can be component used.
+
+{: .image-popup}
+![Screenshot -- Transformations page](/extend/component/code-patterns/overview-0-dev-portal.png)
+
+## New Transformation with Code Pattern

```

Comment:
odsud dal bych to cely dal na help.keboola.com
ten uvod co zbyde pak vratit zpet na /extend/component/code-patterns/overview

Reply:
ok, robim na tom.